### PR TITLE
Small Jekyll YAML Metadata package

### PR DESCRIPTION
**Pull request by nullnein**
Nothing much but I use the command nearly every day. Thought some other users may like it. Makes writing in an editor like iA writer much easier.

### CODE REVIEW

File: packages/jekyll-yaml/0.1.0/_manifest.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name: "jekyll yaml metadata"
+title: "Jekyll YAML metadata"
+description: A simple package to add metadata to your jekyll posts before writing.
+version: 0.1.0
+author: mathis
+tags: ["jekyll", "yaml"]
```
Comment (smeech) [inline packages/jekyll-yaml/0.1.0/_manifest.yml:6]: Would you kindly add a `homepage:` key and value, please, even if the latter is simply https://github.com/nullnein/espanso-hub/tree/main?
We've had problems fully merging packages where a homepage: is missing.

File: packages/jekyll-yaml/0.1.0/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+author: YourName
+tags:
+category:
+title:
+---
+```
+
+Date is set to the current date and authorname can obviously be adjusted.
+
+Please make sure to add tags like the following and remain consistent:
```
Comment (smeech) [inline packages/jekyll-yaml/0.1.0/README.md:17]: I assume this self-evident to Jekyll users? Would it be worth adding a few words about _where_ to add the tags?

File: packages/jekyll-yaml/0.1.0/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+This package lets you add some yaml-metadata to a jekyll post.
+
+Typing `:newpost` yields the following:
+
+```
+---
+date: 2024-12-05
+author: YourName
```
Comment (smeech) [inline packages/jekyll-yaml/0.1.0/README.md:8]: How about adding a note about editing the name value in line 8 of `package.yml`, for Espanso newbies? Or make it a `global_vars:` value nearer the top of the file?

**Review by smeech — CHANGES_REQUESTED** [review]
A single trigger for Jekyll users. No `script`, `shell` or potentially malicious code.

I have a couple of suggestions - the `_manifest.yml` one is particularly important, I'm afraid! Otherwise it's good to go.

File: packages/jekyll-yaml/0.1.0/_manifest.yml (hunk 1 above)
Comment (nullnein) [reply packages/jekyll-yaml/0.1.0/_manifest.yml:6]: sure, done. This should probably be added to the Docs/the template files ;-).

File: packages/jekyll-yaml/0.1.0/README.md (hunk 2 above)
Comment (nullnein) [reply packages/jekyll-yaml/0.1.0/README.md:17]: I may add some more functionality through forms later but I think this basic thing works just fine for now. Good idea, though

File: packages/jekyll-yaml/0.1.0/README.md (hunk 3 above)
Comment (nullnein) [reply packages/jekyll-yaml/0.1.0/README.md:8]: that makes it much prettier, thanks and done

File: packages/jekyll-yaml/0.1.0/_manifest.yml (hunk 1 above)
Comment (smeech) [reply packages/jekyll-yaml/0.1.0/_manifest.yml]: Yes, it's in progress https://github.com/espanso/website/pull/78

**Review by smeech — APPROVED** [review]

**Comment by smeech** [comment]
Possible problem: https://github.com/espanso/hub/actions/runs/12186089096/job/33993784956

**Comment by smeech** [comment]
@arabello?

**Comment by arabello** [comment]
It looks like no releases are found for the repository. The same command the CI runs also returns `no release fund` locally. Also, the static link `https://github.com/espanso/hub/releases/download/v1.0.0/package_index.json` used by the Hub Frontend returns no data.

I need to investigate further, I can't understand right away as I am not confident with espanso/hub. Did some major change happened recently?

**Comment by smeech** [comment]
Would that be in https://github.com/espanso/hub-frontend?
@AucaCoyan updated some versions in the CI recently, but nothing else I can see.

**Comment by AucaCoyan** [comment]
Yup, I deleted the release this afternoon thinking it wasn't used, and after some debugging in this problem I found that was the reason for the failure.
I restored the release, ran the CI job again and everything went fine!

**Comment by smeech** [comment]
Thank you - successfully uploaded to the Hub.

**Pull request merged**